fluent-package/templates/package-scripts/fluent-package/deb/preinst

```diff
@@ -10,40 +10,19 @@ set -e
 # for details, see https://www.debian.org/doc/debian-policy/ or
 # the debian-policy package.
 
-local_base_plugins=/tmp/<%= package_dir %>/.local_base_plugins
-migrate_local_plugins() {
-    if [ "$FLUENT_PACKAGE_SERVICE_RESTART" != auto ]; then
-        return
+check_auto_restart() {
+    if [ -f "/tmp/<%= package_dir %>/.plugin_list" ]; then
+        cp "/tmp/<%= package_dir %>/.plugin_list" "/tmp/<%= package_dir %>/.previous_plugin_list"
```

Comment:
Could you put a proper explanation "why" copy happen here?
(it might be enough to copy it from commit message)

How about name `mark_auto_restart_ready()` because let it in postinst stage.

Reply:
Thanks for your review!
I see!

Fixed: https://github.com/fluent/fluent-package-builder/compare/0b673de8ec94dc360ed5df215b1af8f1c3749568..be1f3fe0d4e568809aa938e46f3444a94fe0ca42